src/main/java/software/aws/neptune/jdbc/DatabaseMetaData.java

```diff
@@ -776,7 +776,7 @@ public int getMaxStatements() {
 
     @Override
     public String getURL() throws SQLException {
-        return "";
+        return this.connection.getClientInfo("contactPoint");
```

Comment:
Why "contactPoint"?

Reply:
contactPoint is the property name used for the host name in ClientInfo

Reply:
Please add a comment as to that point, right now it's a magic constant.